vllm/attention/ops/triton_flash_attention.py

```diff
@@ -312,6 +324,12 @@ def attn_fwd(
     V,
     bias,
     sm_scale,
+    q_scale,
```

Comment:
Is it possible to mark these as constexprs?

Reply:
(discussed offline)